Makefile

```diff
@@ -111,5 +111,12 @@ functest:
 	./hack/build/functests.sh
 
 functest-image-host: WHAT=tools/cdi-func-test-file-host-init
-functest-image-host:  manifests build
+functest-image-host:  manifests docker
```

Comment:
Why build all project binaries and images from scratch when we build the image server?

Reply:
This also creates redundant work for line L115 `./hack/build/build-docker.sh "tools/cdi-func-test-file-host-init tools/cdi-func-test-file-host-http"`